src/main/java/org/team1540/robot2019/subsystems/LEDs.java

```diff
@@ -0,0 +1,35 @@
+package org.team1540.robot2019.subsystems;
+
+import edu.wpi.first.wpilibj.command.Subsystem;
+
+import static org.team1540.robot2019.Hardware.redLEDs;
+import static org.team1540.robot2019.Hardware.blueLEDs;
+
+public class LEDs extends Subsystem {
+
+    public void red() {
```

Comment:
Make these verbs/verb phrases

Better yet, have one method `setColor` that takes an enum.